.gitlint

```diff
@@ -1,6 +1,7 @@
 [general]
-extra-path=scripts/gitlint/rules.py
+extra-path=scripts/gitlint/rules
 regex-style-search=true
+ignore=body-max-line-length
```

Comment:
Won't this allow commits like this - https://github.com/cloud-hypervisor/cloud-hypervisor/commit/7f47a030e72d7aca6897f433b156044ad3da6545 through? Catching commits like this was the original intent of the linter (otherwise we need to back to reviewing every commit message in detail. :-()

Reply:
It is replaced by the Python rule. In my short testing, it should work as expected.

Reply:
Ah, great - thanks!